ONSdigital/eq-prototypes
# Eq 1318 header amends

Amendments to header and introduction page for surveys.

### How to test this 

- Visit `localhost:4000/eq-prototypes/mbs/0111/setup/`
- Enter a company name
- View the intro page.
- View question pages.
- Check that they match the visuals on trello card `1318-company-information-in-eq`
